src/citrine/resources/material_template.py

```diff
@@ -84,9 +84,9 @@ def _build_child_objects(cls, data: dict, session: Session = None):
 
         Returns
         -------
-        dict
-            The serialized material template, but its properties are now a list of object pairs
-            of the form [PropertyTemplate, Bounds].
+        None
+            The serialized measurement template is modified so that its
```

Comment:
```suggestion
            The serialized material template is modified so that its
```
You changed material -> measurement, but this is the material_template file.